assets/js/translator.js

```diff
@@ -132,6 +133,12 @@ if(modeEnabled('translation')) {
             }, timeout);
 
             synchronizeTextareaHeights();
+
+            if(isURL($('#originalText').val())) {
+                setTimeout(function () {
+                    translateWebpage();
```

Comment:
I'm also moderately concerned that we're setting multiple timers. We should ensure that they don't conflict.